inc/class-book.php

```diff
@@ -80,7 +80,7 @@ static function isBook() {
 	 *
 	 * @return array
 	 */
-	static function getBookInformation( $id = null, $contributors_as_string = true ) {
+	static function getBookInformation( $id = null, $contributors_as_string = true, $read_contributors_from_cache = true ) {
```

Comment:
This extra parameter add the flexibility to read or not contributors from cache. If `contributors_as_string` was true from a previous call in this method, then the book information was cached and the contributors information was stored with basic info: only `name`.
With this change, if we set `$read_contributors_from_cache = false` allows dynamically read the contributors from the database, even if the contributors were read as a string and stored in the wp cache.